extensions/metadata/interface/src/main/java/org/exist/storage/md/xquery/Check.java

```diff
@@ -55,8 +52,11 @@
 	
 	public final static FunctionSignature signature =
 		new FunctionSignature(
-			new QName("create", MDStorageManager.NAMESPACE_URI, MDStorageManager.PREFIX),
-			"",
+			new QName("check", MetaData.NAMESPACE_URI, MetaData.PREFIX),
```

Comment:
@adamretter it's useful when metadata storage activated on not empty db. This scan db and create records at metadata storage.